slam-ws/src/nav_husky_gazebo/launch/move_base.launch

```diff
@@ -1,20 +1,18 @@
+<?xml version="1.0"?>
 <launch>
 
    <master auto="start"/>
- <!-- Run the map server --> 
-    <!-- <node name="map_server" pkg="map_server" type="map_server" args="$(find my_map_package)/my_map.pgm my_map_resolution"/> -->
-
- <!--- Run AMCL --> 
-    <!-- <include file="$(find amcl)/examples/amcl_omni.launch" /> -->
```

Comment:
I believe these should be uncommented but instead use the `if` attribute w/in the `<node>` tag to specify whether these nodes should be run